sbtc/src/leb128.rs

```diff
@@ -488,7 +516,23 @@ mod proptests {
 
             // Encoding size should never exceed 10 bytes (for u64 maximum)
             prop_assert!(bytes.len() <= 10,
-                "Encoding should be at most 10 bytes for u64");
+                "encoding should be at most 10 bytes for u64");
+
+            // Explicitly verify continuation bits
+            if bytes.len() > 1 {
```

Comment:
nit: since bytes is non empty (maybe we should add that check) we can remove the if guard here, so that we also check the case of a single byte not having the continuation flag (ie, when `bytes.len() == 1`)

Reply:
It's covered by other tests but yeah, no point in explicitly _not_ testing it here [4e637a4](https://github.com/stacks-network/sbtc/pull/1523/commits/4e637a45ef835b0d3ca871e44380875a0918b8ad)